src/doc/src/reference/environment-variables.md

```diff
@@ -239,6 +239,7 @@ corresponding environment variable is set to the empty string, `""`.
 * `CARGO_PKG_REPOSITORY` --- The repository from the manifest of your package.
 * `CARGO_PKG_LICENSE` --- The license from the manifest of your package.
 * `CARGO_PKG_LICENSE_FILE` --- The license file from the manifest of your package.
+* `CARGO_PKG_EDITION` --- The Rust language edition from the manifest of your package.
```

Comment:
This should list the MSRV.

Reply:
Hmm, looks like #14902 and #14404 didn't add MSRV messages.

We can hold off on this for now because this may require formatting changes, etc.